Baikal/PostEffects/ML/denoiser.cpp

```diff
@@ -22,5 +52,150 @@ namespace Baikal
                                                    height,
                                                    input_channels);
         }
+
+        MLDenoiser::MLDenoiser(CLWContext context, Inference::Ptr inference, MLDenoiserInputs inputs)
+                   : m_inference(std::move(inference))
+        {
+            m_context = std::move(std::make_unique<CLWContext>(context));
```

Comment:
`std::move` is not required. And in the next line.

Reply:
Done